src/crypto/crypto_context.h

```diff
@@ -43,13 +44,27 @@ class SecureContext final : public BaseObject {
 
   const SSLCtxPointer& ctx() const { return ctx_; }
 
+  template <typename Func = void(SSLCtxPointer&)>
+  inline void Initialize(Func fn) {
+    fn(ctx_);
+  }
```

Comment:
I can't figure out where this is used.

Reply:
This may have been a leftover from a prior iteration before I cleaned things up.

Reply:
Handled in #45912